core/src/events.rs

```diff
@@ -22,7 +22,7 @@ use crate::CollisionLayers;
 ///     }
 /// }
 /// ```
-#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Since we are at it, also remove `PartialEq`. I don't think there is any good use case for equality check on that type. And we anyway make a breaking change, so better to promise as little as possible, so that we don't need to make a breaking change next time we add something in this struct.

Reply:
I also removed it from [CollisionData](https://github.com/tqwewe/heron/commit/1113d0f716583359308c05c5a0dd1c13bd959364#diff-4021b9c8437fa077093349ecb73b9e8026433c9d1b64578acd0e406c107bfc10L36) to match the derives with `CollisionEvent`. If you want `CollisionData` to actually derive `PartialEq` let me know and I can add it back.